core/src/main/scala/org/apache/spark/ExecutorAllocationClient.scala

```diff
@@ -81,6 +82,45 @@ private[spark] trait ExecutorAllocationClient {
     countFailures: Boolean,
     force: Boolean = false): Seq[String]
 
+  /**
+   * Request that the cluster manager decommission the specified executors.
+   * Default implementation delegates to kill, scheduler must override
+   * if it supports graceful decommissioning.
+   *
+   * @param executorsAndDecominfo identifiers of executors & decom info.
+   * @param adjustTargetNumExecutors whether the target number of executors will be adjusted down
+   *                                 after these executors have been decommissioned.
+   * @return the ids of the executors acknowledged by the cluster manager to be removed.
+   */
+  def decommissionExecutors(
+    executorsAndDecomInfo: Seq[(String, ExecutorDecommissionInfo)],
```

Comment:
@holdenk, the problem is here:

```
[error] /home/runner/work/spark/spark/core/target/java/org/apache/spark/ExecutorAllocationClient.java:23: error: @param name not found
[info] Generating /home/runner/work/spark/spark/target/javaunidoc/org/apache/spark/ExceptionFailure.html...
[error]    * @param executorsAndDecominfo identifiers of executors &amp; decom info.
[error]             ^
```

I assume there's a bug in Unidoc that fails to generate a proper Javadoc from the Scaladoc. Since this isn't an API, you can either try to exclude this at https://github.com/apache/spark/blob/fa184c3308fcc598c0ff873d606a65d0ad987654/project/SparkBuild.scala#L828-L853

or try to figure out the way to work around, for example, trying different signatures that existent in Java side such as` Array.`

Reply:
Huh weird, I'm confused why I didn't have this show up locally.